.circleci/config.yml

```diff
@@ -180,6 +180,33 @@ jobs:
       - *store_test_package_artifacts
       - *store_package_artifacts
 
+  uat_tests:
+    <<: *container_config
+    steps:
+      - checkout
+      - *restore_rebar_cache
+      - *restore_build_cache
+      - run:
+          name: UAT Tests
+          command: |
+            epmd -daemon
+            make python-env && make swagger && make multi-build
+            make python-uats
```

Comment:
Who does the start of the nodes? I would expect a `make multi-start` somewhere or equivalent.

Reply:
See above comment

Reply:
@velzevur can you bring some light here?

Reply:
In each test we prepare a specific sys.config and start nodes using it. Nodes shouldn't be running before starting the tests.